terminal/lib/features/product/domain/entity/product.dart

```diff
@@ -1,11 +1,35 @@
 import 'package:equatable/equatable.dart';
 
 class Product extends Equatable {
+  final int id;
```

Comment:
Products have a UUID identifier, please use [uuid](https://pub.dev/packages/uuid) instead.
This change has several side effects on the code, be careful!